libkbfs/conflict_resolver.go

```diff
@@ -588,25 +588,19 @@ type createMapKey struct {
 // the given file.  It will return an error if called with a pointer
 // that doesn't represent a file.
 func (cr *ConflictResolver) addChildBlocksIfIndirectFile(ctx context.Context,
-	lState *lockState, original BlockPointer, unmergedChains *crChains,
-	currPath path, op op) error {
-	mostRecent, err := unmergedChains.mostRecentFromOriginalOrSame(original)
-	if err != nil {
-		return err
-	}
+	lState *lockState, unmergedChains *crChains, currPath path, op op) error {
 	// For files with indirect pointers, and all child blocks
 	// as refblocks for the re-created file.
-	fblock, err := cr.fbo.blocks.GetFileBlockForReading(ctx, lState,
-		unmergedChains.mostRecentChainMDInfo.kmd,
-		mostRecent, currPath.Branch, currPath)
+	infos, err := cr.fbo.blocks.GetIndirectFileBlockInfos(
```

Comment:
note to self: we're only using `fileData.getter` here, so there's no need to use `ConflictResolver`'s `newFileData` for its caching behavior.